### PR TITLE
Reduce feed dependencies

### PR DESCRIPTION
**Pull request by tmeschter**
Reduce our NuGet feeds down to a single one, dotnet-project-system-public-packages. This is the same as the one used by the dotnet/project-system repo.

### CODE REVIEW

File: NuGet.config [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
     <add key="dotnet-eng" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json" />
-    <add key="dotnet-tools" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json" />
-    <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" />
-    <add key="dotnet-core" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
+    <add key="dotnet-project-system-public-packages" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/dotnet-project-system-public-packages/nuget/v3/index.json" />
   </packageSources>
-  <disabledPackageSources />
-</configuration>
+  <disabledPackageSources>
+    <clear />
```
Comment (adamint) [inline NuGet.config:10]: is this necessary?

**Review by adamint — APPROVED** [review]

**Comment by adamint** [comment]
@tmeschter i think you will need to add the nuget authenticate task to the ci pipeline.

File: NuGet.config (hunk 1 above)
Comment (smitpatel) [reply NuGet.config:10]: Normal pattern so user who has disabled sources doesn't mess with build for the repo.

File: NuGet.config [hunk 2]
```diff
@@ -1,12 +1,12 @@
-<?xml version="1.0" encoding="utf-8"?>
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
+
 <configuration>
   <packageSources>
     <clear />
-    <add key="myget-legacy" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/myget-legacy/nuget/v3/index.json" />
     <add key="dotnet-eng" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json" />
```
Comment (smitpatel) [inline NuGet.config:6]: This is upstream for dotnet-project-system-public-packages and should no longer needed.

**Comment by tmeschter** [comment]
> @tmeschter i think you will need to add the nuget authenticate task to the ci pipeline

Ohhhh ok. Yeah I'll take a look at that. Thanks!

**Comment by tmeschter** [comment]
OK, I didn't need the NuGet authentication; I just needed to add a bunch of feeds... Now to see if I can cut those down.

File: NuGet.config (hunk 2 above)
Comment (tmeschter) [reply NuGet.config:6]: Turns out I only need dotnet-eng, dotnet-tools, and dotnet-public.

**Pull request merged**